### DISCUSSION TITLE
Unable to Login for Sunbird  Release 5.1 APK

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#723] [General Discussion]
Hello Team,

We are trying to login in to Sunbird release apk 5.1. 
On adding the user id, and password we are getting issue as: 

![image](https://github.com/Sunbird-Ed/Community/assets/72926048/aa900c60-04ff-43e7-9e49-b07b15886f46)
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/2f0df175-2618-4bbe-b979-5d8c61dda22b)

cc: @pallakartheekreddy @krgauraw @@rajeevsathish @mohitga @vinukumar-vs @ashokreddy1208 @swayangjit

**Comment by swayangjit** [comment]
@siddhishinde0723  is portal  release-5.1.0 deployed in your env ? Also post your request and response

**Reply by siddhishinde0723** [reply]
@swayangjit yes the portal is deployed in release - 5.1.0

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

We hope your query is resolved. If yes, could you please close this from your end?

cc: @swayangjit

**Comment by siddhishinde0723** [comment]
Hi @swayangjit 
I am still facing this issue can connect

**Comment by siddhishinde0723** [comment]
Hi Team ,

I tried debugging and found that the session is not set.
Here are few logs for the same 

![image (5)](https://github.com/Sunbird-Ed/Community/assets/72926048/c382e535-7b8f-4078-8733-3a6dce2c2d0b)
![image (4)](https://github.com/Sunbird-Ed/Community/assets/72926048/51147dbf-5209-4136-87d0-adff83ddccf7)


cc: @pallakartheekreddy @krgauraw @@rajeevsathish @mohitga @vinukumar-vs @ashokreddy1208 @swayangjit

**Comment by HarishGangula** [comment]
@siddhishinde0723  Can you give more context of the issue to suggest for the fix for it.

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
